layouts/_default/search.html

```diff
@@ -29,6 +29,7 @@ <h2 class="post-title no-text-decoration"><a class="search_link search_title" hr
 {{ define "footer" }}
   <!-- If you don't need IE 11 support you can remove this next script block -->
   <script type="text/javascript">
+    // @license magnet:?xt=urn:btih:1f739d935676111cfff4b4693e3816e664797050&dn=gpl-3.0.txt GPL-v3-or-Later
```

Comment:
What is that 1f73.. number? How do you derive that?

Reply:
That is a magnet link the GPL license. You can find those on the page which I linked in my first comment. 
- https://www.gnu.org/software/librejs/free-your-javascript.html#magnet-link-license